src/services/analytics/useGtm.ts

```diff
@@ -34,13 +47,14 @@ const useGtm = () => {
   // Track page views – anononimized by default.
   // Sensitive info, like the safe address or tx id, is always in the query string, which we DO NOT track.
```

Comment:
Should we also update this comment or are we still anonymizing these?

Reply:
We're still anonymizing these.